### ISSUE TITLE
Finetune Task For Code Completion

### ISSUE THREAD

**Issue by HoratioJSY** [salesforce/CodeT5#12]
Hi, Is there a fine-tune task for code completion? Maybe, I can implement Masked Span Prediction Task as fine-tune task for code generation (like gpt style)?

**Comment by yuewang-cuhk** [comment]
Hi, we do not fine-tune CodeT5 on code completion. You might refer to [this paper](https://arxiv.org/abs/2105.05991) for how to apply seq2seq models for code autocompletion task.
